src/backend/CassandraBackend.cpp

```diff
@@ -1391,8 +1417,9 @@ CassandraBackend::open(bool readOnly)
               << "issuer_nf_tokens"
               << "  ("
               << "    issuer blob,"
+              << "    token_taxon bigint,"
               << "    token_id blob,"
-              << "    PRIMARY KEY (issuer, token_id)"
+              << "    PRIMARY KEY (issuer, token_taxon, token_id)"
```

Comment:
So now that we are changing this, what happens if the table already exists? Does clio crash? We probably need some easy way for people to update their database. Maybe a simple script?

Reply:
We need to support migrations.. it's kinda a bit late tho. Any good DB wrapper/library supports migrations in one way or the other. Worth checking if datastax provides anything at all in their cpp library. If they don't then we need a new table that will hold the versions of each table and some support in code to apply changes to the table structure when migrating from version X to version Y. and for each update of the table we increment the version and add a migration.

Reply:
We do, though I'm not sure if a db wrapper will always be able to do what we want. There are things that would require preprocessing the ledger data in a way the dbwrapper cannot do itself. Do you think a db wrapper would be able to do what we have done in this PR? It seems to me that we just need to rerun ETL here.

Reply:
I'm not sure i understand your concern. If you support migrations you can run arbitrary queries (and c++ code) against the DB when launching first time with DB of version XXX and `clio` that needs DB of version YYY. So you can literally do anything with the DB and/or run any C++ code you need to support the migration.

Reply:
oh i see, I misunderstood. yes if the dbwrapper can just run arbitrary c++ code, we can probably support arbitrarily complex migrations, at least anything that can be done solely with the data already in the database. I was thinking the migration would need to happen exclusively server side on the database itself.

Reply:
see script above

Reply:
the script doesn't have a concept of versions, maybe we can add that in